packages/core/src/server/service.test.ts

```diff
@@ -1575,3 +1576,74 @@ test.skip("serves derived entities versioned at provided timestamp", async (cont
 
   await service.kill();
 });
+
+// Admin routes.
+
+describe("/admin/reload", async () => {
+  it("emits chainIds in reload event", async (context) => {
```

Comment:
Style nit: We prefer top-level `test` statements with longer descriptions over `describe`/`it` blocks. I've found that nesting tests can get unwieldy very quick (see https://kentcdodds.com/blog/avoid-nesting-when-youre-testing).

So these would just be
```ts
test("/admin/reload emits chainIds in reload event", async (context) => {
  // ...
})
```

Totally fine if the first few words of the description are repeated many times.